projects/demo/src/modules/components/select/examples/11/index.html

```diff
@@ -0,0 +1,27 @@
+<form class="b-form">
+    <tui-select [formControl]="itemControl">
+        Character
+        <select
+            tuiSelect
+            [items]="items"
+        ></select>
+    </tui-select>
+
+    <tui-select
+        class="tui-space_top-4"
+        [formControl]="itemGroupControl"
+        [tuiTextfieldCleaner]="true"
+    >
+        Food
+        <select
+            tuiSelect
+            [items]="groupItems"
+            [labels]="labels"
```

Comment:
Dublicated group names.
Is it safari-browser bug ?

![safari-dublicated-group-names](https://user-images.githubusercontent.com/35179038/198982479-36e71ebb-3a89-484e-a8da-b10983ef9bb5.jpg)

Reply:
Well, html is straightforward. Do you see duplicated groups here?
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/optgroup

Reply:
yes)

![photo_2022-10-31_13-22-03](https://user-images.githubusercontent.com/35179038/198986390-c095c4e0-696d-4cea-a612-88d74737788a.jpg)

Reply:
https://stackoverflow.com/questions/73903396/ios-16-shows-optgroup-tags-label-two-times

https://bugs.webkit.org/show_bug.cgi?id=247204

Reply:
ios >16 only